bin/zowe-support.sh

```diff
@@ -88,16 +88,22 @@ add_to_pax $PS_OUTPUT_FILE process_info
 rm $PS_OUTPUT_FILE
 
 # Collect STC output
-STC_JOBS=`tsocmd "STATUS ${ZOWE_STC}" 2>/dev/null | grep -E 'OUTPUT' | cut -d' ' -f2`
-for STC in ${STC_JOBS}
+if [ -z ${TEMP_DIR+x} ]; then
```

Comment:
I think this if should be removed. TEMP_DIR doesn't exist at this point

Reply:
Done.

Reply:
@John-A-Davies - when is the commit that you've done this in going to be pushed? I think it's all that is holding the merge up

Reply:
I think I removed it this time.